trade-finance-manager-api/src/v1/rest-mappings/mappings/gef-facilities/mapGefFacility.js

```diff
@@ -28,6 +33,7 @@ const mapGefFacility = (facility, dealSnapshot, dealTfm) => {
     _id: facility._id,
     facilitySnapshot: {
       _id: facility._id,
+      isGef: true,
```

Comment:
fwiw this is moved to snapshot as it's a computed property of the snapshot. the snapshot should not be modified, but other functions are heavily modifying existing values (and not just adding a new computed parameter).